### PR TITLE
D   Separate risk from intention

### PR DESCRIPTION
**Pull request by arlobelshee**
Reworks nearly the entire file & notation. Defines 4 levels of risk (lower-case, upper-case, !!, and **). Also defines a set of core intentions (feature, bugfix, refactor/remodel, and documentation). Each intention can conceivably be performed at each level of risk. Shows examples of how to attain a given risk level for each core intention. Finally, includes several extended intentions. Projects may or may not use these; each provides alternatives if the project chooses to not use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,128 @@
...
+## Extension Intentions
+
+Each project can define a set of extension intentions. Each project should define which extension codes it uses. It is up to each project to define the approaches for each of the 4 risk levels.
+
+These are some common intentions, each used in several projects. Each also lists alternatives used in projects that don't use the code.
+
+| Prefix | Name | Intention | Alternatives |
+| --- | --- | --- | --- |
+| M | Merge | Merge branches | Use `F`, `B`, or `R`, based on the main intention of the branch, with risk level based on maximum for any individual commit in the branch. Optionally leave blank for merge from main to a feature branch. |
+| T | Test-only | Alter automated tests without altering functionality. May include code-generating code that just throws a `NotImplementedException` or similar approaches. | Use `f` or `b`, depending on which kind of work this test is going to validate. It is a lower-case letter unless you also change product code. |
```
Comment (JayBazuzi) [inline README.md:119]: What do you think of using `r` when refactoring entirely within test code?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,128 @@
...
-| F!!     | Feature (> 8 <abbr title="lines of code">LoC</abbr><sup>[4]</sup>)                             |
-| B!!     | Bug (> 8 <abbr title="lines of code">LoC</abbr><sup>[4]</sup>)                                   |
-| R!!     | Non-provable refactoring                                  |
-| ***     | Does not compile intermediate step                        |
+## The Four Risk Levels
 
-# Description
+| Risk Level | Code | Example | Meaning |
+| --- | --- | --- | --- |
+| **Known safe** | lowercase letter | `r   Extract method Applesauce` | Addresses all known and unknown risks. |
```
Comment (JayBazuzi) [inline README.md:19]: It a lot of UI `   ` doesn't format as three spaces, it collapses to one. I wish for a notation that doesn't have that problem.

In one project we use `r - ` instead. Maybe riff on that?

This problem may be out of scope for the current pull request.

Comment (JayBazuzi) [reply README.md:19]: image

File: README.md (hunk 1 above)
Comment (arlobelshee) [reply README.md:119]: I like it. I'll add that as another known way to be at that level, as well as an alternative to t.

File: README.md (hunk 2 above)
Comment (arlobelshee) [reply README.md:19]: That's a good idea. I think it warrants its own PR / discussion.

Comment (JayBazuzi) [reply README.md:19]: https://github.com/RefactoringCombos/ArlosCommitNotation/issues/26

**Pull request merged**